projects/framework-blender/extensions/plugins/blender_scene_saved_validator.py

```diff
@@ -0,0 +1,82 @@
+# :coding: utf-8
+# :copyright: Copyright (c) 2024 ftrack
+
+import bpy
+
+from ftrack_utils.paths import get_temp_path
+
+from ftrack_framework_core.plugin import BasePlugin
+from ftrack_framework_core.exceptions.plugin import (
+    PluginExecutionError,
+    PluginValidationError,
+)
+
+
+class BlenderSceneSavedValidatorPlugin(BasePlugin):
+    name = 'blender_scene_saved_validator'
+
+    def save_to_temp(self, store, extension_format):
+        '''
+        Save the file to a temporary location.
+        '''
+
+        try:
+            # Save file to a temp file
+            save_path = get_temp_path(filename_extension='.blend')
+            # Save Blender scene to this path
+            bpy.ops.wm.save_as_mainfile(filepath=save_path)
+            self.logger.debug(f"Scene has been saved to: {save_path}.")
+        except Exception as error:
+            raise PluginExecutionError(message=error)
+
+        component_name = self.options.get('component', 'main')
+        store['components'][component_name]['scene_name'] = save_path
+        store['components'][component_name]['scene_saved'] = True
+        store['components'][component_name]['valid_file'] = True
+
+    def save_scene(self, store):
+        '''
+        Save the current blender scene.
+        '''
+        bpy.context.scene.render.filepath = None
+
+        try:
+            save_path = get_temp_path(filename_extension='.blend')
+            bpy.ops.wm.save_as_mainfile(filepath=save_path)
+            self.logger.debug(f"Scene has been saved.")
+        except Exception as error:
+            raise PluginExecutionError(
+                message=f"Error saving the scene: {error}"
+            )
+
+        component_name = self.options.get('component', 'main')
+        store['components'][component_name]['scene_saved'] = True
+        store['components'][component_name]['valid_file'] = True
```

Comment:
This is doing the same as save_to_temp, it shouldn't save to temp in this one. Please fix it or remove it if not used.